koku/sources/storage.py

```diff
@@ -278,6 +290,7 @@ def enqueue_source_delete(source_id, offset, allow_out_of_order=False):
         if not source.pending_delete and not source.out_of_order_delete:
             source.pending_delete = True
             source.save()
+            mark_provider_as_inactive(source.koku_uuid)
```

Comment:
This raises `Provider.DoesNotExist`. Is this exception caught anywhere?

Reply:
Right here: https://github.com/project-koku/koku/pull/2743/files#diff-c25b68ba7a2e6f35a566bf4844d96b05c08ae2b4a72ffb29e99b07118f02bdacR272